mailpoet/lib/Automation/Engine/Engine.php

```diff
@@ -14,6 +14,8 @@
 use MailPoet\Automation\Integrations\Core\CoreIntegration;
 
 class Engine {
+  const CAPABILITY_MANAGE_AUTOMATIONS = 'mailpoet_manage_automations';
```

Comment:
I think this should be fine as-is. It's probably too late to fully change usages of `permission` to `capability` in the free plugin, especially the strings themselves since users are probably relying on those in production sites, but having `capability` in the automation codebase is nice.

I think I'm still getting used to the level of separation we're shooting for with the automation codebase and the plugins.